packages/sdk-bundles-api/package.json

```diff
@@ -17,6 +17,10 @@
     "lint:commit": "eslint --cache $(git diff --relative --name-only --diff-filter=ACMRTUXB master | grep  -E \"(.js$|.ts$|.tsx$)\")",
     "lint:fix": "eslint src --ext=ts,tsx,js --cache --fix"
   },
+  "devDependencies": {
```

Comment:
Are these devDependencies or peerDependencies? (can't remember details of how peer dependencies work / are inherited; but if they do work as desired for this then it could be quite nice having them as peerDependencies, because it'd enforce both main repo & sdk repo to use same version of puppeteer -- using separate versions of puppeteer may break things)

Reply:
Right :+1: